pytest.ini

```diff
@@ -19,13 +19,8 @@
 #    updated.
 # 3) Ignore DeprecationWarning for datetime.utcfromtimestamp() triggered
 #    from dateutil. See https://github.com/dateutil/dateutil/issues/1314.
-# 4 & 5) The pyOpenSSL X509/PKey warnings are due to TLS-ALPN-01 support.
-#        Resolving these warnings is being tracked by
-#        https://github.com/certbot/certbot/issues/10079.
-# 6 - 11) Planning to remove unused TLS-ALPN support in acme.
-#         See https://github.com/certbot/certbot/issues/10266 and
-#         https://github.com/certbot/certbot/pull/10294.
-# 12) Planning to remove support for checking OCSP via OpenSSL binary.
+# 4) Planning to remove SSLSocket
```

Comment:
Is there an issue tracking this? Or reason to not include this removal in this PR?